.github/workflows/ci-build.yaml

```diff
@@ -119,6 +119,20 @@ jobs:
               echo "No .apk files found."
           fi
 
+      - name: Check SBOMs
+        env:
+          NTIA_IMAGE: cgr.dev/chainguard/ntia-conformance-checker:latest@sha256:bde670cb65228e873ca2a2d674d407f310b11e608465454c668a5285fbaba3ee
```

Comment:
What’s the image for?

Reply:
A previous iteration of this did `docker run <image>`, so this isn't needed anymore. Good catch, it's gone.

Approval again pls 🙏